lib/cn/cn_tree.c

```diff
@@ -1139,6 +1135,13 @@ cn_tree_prepare_compaction(struct cn_compaction_work *w)
 
     n_outs = CN_FANOUT_MAX;
```

Comment:
Why do we still need n_outs to be CN_FANOUT_MAX for spills?

Reply:
We don't. I can make that zero.

Reply:
Split is the only operation that makes multiple kvsets and it tracks them in other struct members.  So maybe `struct kvset_mblocks    *cw_outv` can be changed to `struct kvset_mblocks cw_mblocks` or something?

Reply:
Split still relies on cw_outv. So this can't be changed until we change split code to use a separate struct member in w.cw_split for outv.